TEI.js

```diff
@@ -1,25 +1,25 @@
 {
-	"translatorID": "032ae9b7-ab90-9205-a479-baf81f49184a",
-	"translatorType": 2,
-	"label": "TEI",
-	"creator": "Stefan Majewski",
-	"target": "xml",
-	"minVersion": "4.0.27",
-	"maxVersion": null,
-	"priority": 25,
-	"inRepository": true,
-	"configOptions": {
-		"dataMode": "xml/dom",
-		"getCollections": "true"
-	},
-	"displayOptions": {
-		"exportNotes": false,
-		"Export Tags": false,
-		"Generate XML IDs": true,
-		"Full TEI Document": false,
-		"Export Collections": false
-	},
-	"lastUpdated": "2016-01-19 20:45:00"
+    "translatorID": "032ae9b7-ab90-9205-a479-baf81f49184a",
+    "translatorType": 2,
+    "label": "TEI",
+    "creator": "Stefan Majewski",
+    "target": "xml",
+    "minVersion": "4.0.27",
+    "maxVersion": null,
+    "priority": 25,
+    "inRepository": true,
+    "configOptions": {
+        "dataMode": "xml/dom",
+        "getCollections": "true"
+    },
+    "displayOptions": {
+        "exportNotes": false,
+        "Export Tags": false,
+        "Generate XML IDs": true,
+        "Full TEI Document": false,
+        "Export Collections": false
+    },
+    "lastUpdated": "2016-01-22 20:45:00"
```

Comment:
Yep, screwed up the date. I copy/pasted what came out of beautify with the default settings directly in -- what settings should I use? Indent with a tab character?